src/Service.Tests/SqlTests/SqlTestHelper.cs

```diff
@@ -181,16 +201,7 @@ public static void TestForErrorInGraphQLResponse(string response, string message
                 JsonElement expectedErrorObj = JsonDocument.Parse(expected).RootElement.GetProperty(PARENT_PROPERTY_ERROR);
                 string expectedStatusCode = expectedErrorObj.GetProperty(PROPERTY_STATUS).ToString();
                 string expectedSubStatusCode = expectedErrorObj.GetProperty(PROPERTY_CODE).ToString();
-
-                // Quote(") has to be treated differently than other unicode characters
-                // as it has to be added with a preceding backslash.
-                responseBody = Regex.Replace(responseBody, @"\\u0022", @"\\""");
-
-                // Remove all carriage returns and new lines from the response body.
-                responseBody = Regex.Replace(responseBody, @"\\n|\\r", "");
-
-                // Convert the escaped characters into their unescaped form.
-                responseBody = Regex.Unescape(responseBody);
+                responseBody = PreprocessResponse(responseBody);
```

Comment:
Any reason for moving this into a function now? Its good, but just curious...

Reply:
`Currently GQL does not have any tests (but REST has) which needs this replacement logic on the exception message. In future, GQL can also utilize this method and looks cleaner now.